.circleci/config.yml

```diff
@@ -251,21 +251,6 @@ jobs:
 
     - run: ./scripts/circleci/run-saucelabs-tests.sh
 
-  # -----------------------------------------------------------------------------------------
-  # Job that builds the dev-app with AOT. In order to speed up this job, the release output
-  # from the workspace storage will be attached to this job.
-  # -----------------------------------------------------------------------------------------
-  build_devapp_aot:
```

Comment:
Should we still build the devapp as a test?

Reply:
It will be built as part of the bazel build job.